### DISCUSSION TITLE
Hey! I can't join my team on vercel coz signup and login has error, even I never created acct and it's my first time to vercel! I am urgent a little bit!

### DISCUSSION THREAD

**Discussion by bmcpaul02** [vercel/community#4726] [Account Issue]
### Question

Hey! I can't join my team on vercel coz signup and login has error, even I never created acct and it's my first time to vercel! I am urgent a little bit!

**Comment by amyegan** [comment]
Sorry you ran into trouble signing up, @bmcpaul02. We can't safely discuss account details here in public. But if you already used one of the [direct channels](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) to open a case, I can check on it for you. Just let me know the case number 🙏

**Reply by bmcpaul02** [reply]
00163315

BTW I don't remember how I opened this issue?
Anyway please email me after you check this case number.
Thanks!
